pandas/core/series.py

```diff
@@ -1038,12 +1038,61 @@ def _set_values(self, key, value):
 
     def repeat(self, repeats, *args, **kwargs):
         """
-        Repeat elements of an Series. Refer to `numpy.ndarray.repeat`
-        for more information about the `repeats` argument.
+        Repeat elements of a Series.
```

Comment:
any way to re-use the doc-string you defined for EA (meaning here and for Index), maybe make it even more generic and parametrize on the types? (ok that we are duplicating code, but the more can share the better).
cc @datapythonista

Reply:
Had the same thought but was unsure how to do it in a clean way.  I think you could split the docstring to reuse to everything but the See Also and Examples section, then separately define only those sections for EA/Index/Series.  Seems a bit convoluted though; certainly interested to hear if there's a better way.